CMSIS/Core/Include/cmsis_armcc.h

```diff
@@ -875,6 +875,10 @@ __attribute__((always_inline)) __STATIC_INLINE uint32_t __USAT(int32_t val, uint
 #define __SMMLA(ARG1,ARG2,ARG3)          ( (int32_t)((((int64_t)(ARG1) * (ARG2)) + \
                                                       ((int64_t)(ARG3) << 32U)     ) >> 32U))
 
+#define __SXTB16_RORn                    __SXTB16(__ROR(op1, rotate))
```

Comment:
Did you miss the arguments to this function-like macro?
Shouldn't it be `#define __SXTB16_RORn(op1, rotate)` instead?

Reply:
My bad. I forgot to copy the correct definition